envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdaterTest.kt

```diff
@@ -43,13 +47,24 @@ class SnapshotUpdaterTest {
         cache.setSnapshot(AllServicesGroup(ads = false), uninitializedSnapshot)
         cache.setSnapshot(groupWithProxy, uninitializedSnapshot)
         cache.setSnapshot(groupWithServiceName, uninitializedSnapshot)
-        cache.setSnapshot(groupOf(services = listOf("existingService1")), uninitializedSnapshot)
-        cache.setSnapshot(groupOf(services = listOf("existingService2")), uninitializedSnapshot)
+        cache.setSnapshot(groupOf(
+            services = setOf(ServiceDependency("existingService1"))
+        ), uninitializedSnapshot)
+        cache.setSnapshot(groupOf(
+            services = setOf(ServiceDependency("existingService2"))
+        ), uninitializedSnapshot)
+
+        cache.setSnapshot(groupOf(
+            services = setOf(
```

Comment:
This is a similar situation I described elsewhere. The pattern is repeated, while you could just use a function different than `listOf` that accepts these arguments and does the domain objects creation underneath.